README.md

```diff
@@ -79,19 +79,19 @@ compileOptions {
 
 ## Snapshots
 
-[![CI status](https://ci.novoda.com/buildStatus/icon?job=no-player-snapshot)](https://ci.novoda.com/job/no-player-snapshot/lastBuild/console) [![Download from Bintray](https://api.bintray.com/packages/novoda/snapshots/no-player/images/download.svg)](https://bintray.com/novoda/snapshots/no-player/_latestVersion)
+[![CI status](https://ci.novoda.com/buildStatus/icon?job=no-player-snapshot)](https://ci.novoda.com/job/no-player-snapshot/lastBuild/console) [![Download from Bintray](https://api.bintray.com/packages/novoda-oss/snapshots/no-player/images/download.svg)](https://bintray.com/novoda-oss/snapshots/no-player/_latestVersion)
```

Comment:
I guess this is just temporary until we merge `develop` right? Because these same changes have been introduced in https://github.com/novoda/no-player/pull/230/files#diff-04c6e90faac2675aa89e2176d2eec7d8R82

Reply:
yeah, we can't really merge `develop` just yet

Reply:
ok then